playbooks/openstack/sample-inventory/group_vars/all.yml

```diff
@@ -184,19 +184,6 @@ ansible_user: openshift
 # NOTE: this is for testing only! Your data will be gone once the VM disappears!
 # openshift_openstack_ephemeral_volumes: false
 
-# # OpenShift node labels
-# # - in order to customise node labels for app and/or infra group, set the
-# #   openshift_openstack_cluster_node_labels variable
-# # - to enable the multi-pool driver support at Kuryr the driver for the pod
-#  #  pod vif need to be added as a node label
-#openshift_openstack_cluster_node_labels:
```

Comment:
I see you have removed this from all.yml and based on the documentation moved to the OSEv3.yml which is a better place for it. It will be nice to add a commented example about the new way to do it to the sample-inventory/group_vars/OSEv3.yaml as we use to have for the old format in all.yml. What do you think?

Reply:
I'd like to remove most of the custom options from the sample-inventory because it's getting hard too unwieldy (and we have to maintain two sets of changes -- one for the docs and the other for the inventory). But since the info already existed there, I'll add it back.

Reply:
I didn't now that was the intention, but if that so feel free to remove it. I just asked for it since it was easier for me to just copy the inventory and have the information there ready to be uncommented, rather than go to the documentation and copy and paste it (with the high possibility of me forgetting some of the inputs)